examples/AGS02MA_calibrate/AGS02MA_calibrate.ino

```diff
@@ -1,59 +1,133 @@
 //
 //    FILE: AGS02MA_calibrate.ino
-//  AUTHOR: Rob Tillaart
-// VERSION: 0.1.0
+//  AUTHOR: Rob Tillaart, Beanow
+// VERSION: 0.2.0
 // PURPOSE: test application
 //    DATE: 2021-08-12
 //     URL: https://github.com/RobTillaart/AGS02MA
 //
 
 #include "AGS02MA.h"
 
+// You can decrease/disable warmup when you're certain the chip already warmed up.
+#define WARMUP_MINUTES 6
+#define READ_INTERVAL 3000
 
-uint32_t start, stop;
 
+uint32_t start, stop;
+u_int8_t version;
 
 AGS02MA AGS(26);
 
 
 void setup()
 {
+  // ESP devices typically mis the first serial log lines after flashing.
+  // Delay somewhat to include all output.
+  delay(1000);
+
   Serial.begin(115200);
```

Comment:
Some platforms need a line ```while(!Serial);```  - think the NANO33 (mbed) needs but people can add that theselves.
(no real issue)